app/models/assignment.rb

```diff
@@ -16,7 +16,7 @@ def authorized_for_delete?
 
   def self.add_volunteer_to_region volunteer, region
     return false if volunteer.new_record?
-    if Assignment.where(:volunteer_id=>volunteer, :region_id=>region.id).count==0
+    if Assignment.where(volunteer_id: volunteer, region_id: region.id).count == 0
```

Comment:
Seems that `.count == 0` could be replaced with `.empty?`, but I think it could probably be merged as-is.